documentation/docs/developers/strategies/cross-exchange-market-making.md

```diff
@@ -0,0 +1,96 @@
+
+In Hummingbot code and documentation, we usually refer to the less liquid market as the "maker side" - since the cross exchange market making strategy is providing liquidty there. We then refer to the more liquid market as the "taker side" - since the strategy is taking liquidity there.
+
+The cross exchange market making strategy's code is divided into two major parts:
+
+ 1. Order creation and adjustment
+
+    Periodically creates and adjusts limit orders on the maker side.
+ 
+ 2. Hedging order fills
+
+    Performs the opposite, hedging trade on the taker side, whenever a maker order has been filled.
+
+## Order Creation and Adjustment
+
+Here's a high-level view of the logical flow of the order creation and adjustment part. The overall logic of order creation and adjustment is pretty involved, but it can be roughly divided to the **Cancel Order Flow** and the **Create Order Flow**.
+
+The cross exchange market making strategy regularly refreshes the limit orders it has on the maker side market by regularly cancelling old orders (or waiting for existing order to expire), and creating new limit orders. This process ensures the limit orders it has on the maker side are always of the correct and profitable prices.
+
+![Figure 1: Order creation and adjustment flow chart](/assets/img/xemm-flowchart-1.svg)
+
+The entry point of this logic flow is the `c_process_market_pair()` function in [`cross_exchange_market_making.pyx`](https://github.com/CoinAlpha/hummingbot/blob/master/hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making.pyx).
+
+### Cancel Order Flow
+
+The cancel order flow regularly monitors all active limit orders on the maker side, to ensure they are all valid and profitable over time. If any active limit order becomes invalid (e.g. because the asset balance changed) or becomes unprofitable (due to market price changes), then it should cancel such orders.
+
+![Figure 2: Cancel order flow chart](/assets/img/xemm-flowchart-2.svg)
+
+#### Active order cancellation setting
+
+The [`active_order_canceling`](/strategies/corss-exchange-market-making/) setting changes how the cancel order flow operates. `active_order_canceling` should be enabled when the maker side is a centralized exchange (e.g. Binance, Coinbase Pro), and it should be disabled when the maker side is a decentralized exchange.
+
+When `active_order_canceling` is enabled, the cross exchange market making strategy would refresh orders by actively cancelling them regularly. This is optimal for centralized exchanges because it allows the strategy to respond quickly when, for example, market prices have significantly changed. This should not be chosen for decentralized exchanges, because actively cancelling orders on DEXes can often take a long time and may also require gas cost.
```

Comment:
"This should not be chosen for decentralized exchanges that charge gas for cancelling orders (such as Radar Relay
)"